### ISSUE TITLE
Standard Ket Notation in Qiskit Textbook

### ISSUE THREAD

**Issue by priyanshi-git** [qiskit-community/qiskit-textbook#1099]
**Is your feature request related to a problem? Please describe.**
In Qiskit Textbook, the ways to see our quantum state are limited, such as 

1) We can use **Statevector Simulator** to visualize our quantum state but that only give us an array of the amplitude of our possible quantum state
For example, if we have a quantum circuit named qc with only one qubit and we applied qc.i(q[0]) then the result we will get after using get_statevector would be array[(1.+0.j, 0.+0.j)] or,

2) We can use **array_to_latex** to see a column vector having amplitude of the possible quantum states (again).  

Basically there is no way we can actually see the possible state but only their amplitudes.

**Describe the solution you'd like**
So, I suggest that there should be a function that can come handy to beginners a lot to visualize the quantum state to present a Quantum state in standard ket notation 

For example, let us assume the name of that function is "state" and then we have the following code 
```
qc = QuantumCircuit(1)
qc.i(0)

state(qc)
```

Then the output must be `1.0|0>`

**Comment by Abhigyan-Mishra** [comment]
I am willing to help out with this.

**Comment by frankharkins** [comment]
There is someone working on this already in Qiskit/qiskit-terra#6154. I think best course of action is to wait for that to be merged and then take advantage of it in the textbook.

**Comment by frankharkins** [comment]
I'm going to close this because this is being worked on in terra, please reopen if you disagree
